packages/core/core-flows/src/order/workflows/add-line-items.ts

```diff
@@ -31,7 +31,7 @@ import { productVariantsFields } from "../utils/fields"
 
 function prepareLineItems(data) {
   const items = (data.input.items ?? []).map((item) => {
-    const variant = data.variants?.find((v) => v.id === item.variant_id)!
+    const variant = data.variants?.find((v) => v.id === item.variant_id);
```

Comment:
todo: we don't use semi colons
```suggestion
    const variant = data.variants?.find((v) => v.id === item.variant_id)
```